js/data/locale/ar/BH/ctrynames.json

```diff
@@ -292,5 +292,6 @@
     "هولندا الكاريبية": "BQ",
     "هونغ كونغ الصينية (منطقة إدارية خاصة)": "HK",
     "وسط آسيا": "143",
-    "وسط أفريقيا": "017"
+    "وسط أفريقيا": "017",
+    "Sark": "CQ"
```

Comment:
This is not right... Sark has the ISO 3166 code of GG, not CQ. CQ is the Northern Marianas Islands.

Also, the name does not seem to be translated to any of the languages in any of these files.

Reply:
But, this data is newly added from cldr v43. from cldr-localenames-full package.
i.e) https://github.com/unicode-org/cldr-json/blame/main/cldr-json/cldr-localenames-full/main/en-AE/territories.json
Do you think it better filter the data out in the tool code?

Reply:
and the reason the data has not translated case, it's because of the tool code. the tool code does something.
it looks like country code information `CQ` is only added `en-Region` locale and `hi-Latn` only.

Reply:
Regarding the code `CQ`
https://www.iso.org/obp/ui/#iso:code:3166:CQ
> "Previously Sark was represented by Guernsey's country code (GG). Sark also fought for 20 years to get the .cq country code top-level domain."
from Politics https://en.wikipedia.org/wiki/Sark

the chromium has been updated too. https://chromium-review.googlesource.com/c/chromium/src/+/4783452

and It looks like MP is Northern Marianas Island 's ISO code.  (https://en.wikipedia.org/wiki/Northern_Mariana_Islands)